### PR TITLE
pyPopMiner

### PR DESCRIPTION
**Pull request by overcookedpanda**
This PR should fully integration pyPopMiner and the tests should run with each build, however, due to some Boost warnings, we may have to revise to run this particular build without -Werror

### CODE REVIEW

File: pypopminer/CMakeLists.txt [hunk 1]
```diff
@@ -12,19 +12,21 @@ if(NOT ${Boost_FOUND})
 endif()
 find_package(PythonInterp 3)
 find_package(PythonLibs 3)
-add_custom_target(pypopminer COMMENT "Building pypopminer")
+add_custom_target(src COMMENT "Building pyPopMiner")
```
Comment (Warchant) [inline pypopminer/CMakeLists.txt:15]: Why is target named src?

Comment (overcookedpanda) [reply pypopminer/CMakeLists.txt:15]: IDE got carried away.. fixes incoming :)

File: pypopminer/src/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+include_directories(${PROJECT_SOURCE_DIR}/src)
```
Comment (Warchant) [inline pypopminer/src/CMakeLists.txt:2]: Why do you add this?

Comment (overcookedpanda) [reply pypopminer/src/CMakeLists.txt:2]: `set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})` was existing in your original commits, somewhere through the debugging process, I added `include_directories(${PROJECT_SOURCE_DIR}/src)` but it seems both can be removed now.  Will commit fix now.

**Pull request merged**